client/src/app/components/manage-inventory-items/manage-inventory-items.component.html

```diff
@@ -18,8 +20,15 @@
         <td mat-cell *matCellDef="let object">{{ object[column]}}</td>
       </ng-container>
 
-      <tr mat-header-row *matHeaderRowDef="displayedColumns"></tr>
-      <tr mat-row *matRowDef="let row; columns: displayedColumns;"></tr>
+      <ng-container matColumnDef="Select">
+        <th mat-header-cell *matHeaderCellDef>Select</th>
+        <td mat-cell *matCellDef="let object">
+          <mat-checkbox  class="example-margin" (change)='onChange(object["_id"])'></mat-checkbox>
+        </td>
+      </ng-container>
```

Comment:
can we put this column before all the others?

Reply:
Yes we could.